stdlib/VERSIONS

```diff
@@ -50,6 +50,7 @@ _threading_local: 3.6-
 _tkinter: 2.7-
 _tracemalloc: 3.6-
 _typeshed: 2.7-  # not present at runtime, only for type checking
+_typeshed.tkinter: 3.0-
```

Comment:
This is a weird module. We should consider removing it, because it's only used in one file.